mindmeld/system_entity_recognizer.py

```diff
@@ -442,7 +442,11 @@ def resolve_system_entity(self, query, entity_type, span):
             )
 
         if len(entity_type_filtered_candidates) > 0:
-            return entity_type_filtered_candidates[-1]
+            # Duckling sorts most probable entity candidates higher than
+            # the lower probable candidates. So we return the best possible
+            # candidate in this case when multiple duckling candidates are
+            # returned.
+            return entity_type_filtered_candidates[0]
```

Comment:
doesn't this select the wrong candidate for the time grain cases?

Reply:
Do you have an example?